eip-halvening.md

```diff
@@ -0,0 +1,68 @@
+---
+eip: <to be assigned>
+title: Update Network Block Rewards to Adhere to "Minimum Viable Issuance" Policy
+author: Kevin Owocki <kevin@gitcoin.co>
+discussions-to: Pull Request
+status: Draft
+type: Economic
+created: 2020-08-11
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+Ethereum's monetary policy is "Minimum Viable Issuance to secure the network". Now that [miners are nearly collecting more ETH from fees than blockrewards](https://twitter.com/econoar/status/1293220294396538880?s=20) I believe we should cut block rewards from 2 ETH/block to 1 ETH/block.
```

Comment:
EIPs should avoid externally linking to content since it is likely that the EIP will outlast most external content.

```suggestion
Ethereum's monetary policy is "Minimum Viable Issuance to secure the network". Now that miners are nearly collecting more ETH from fees than block rewards I believe we should cut block rewards from 2 ETH/block to 1 ETH/block.
```